packages/webui/src/client/lib/logStatus.ts

```diff
@@ -12,7 +11,7 @@ import { logger } from './logging'
 const browserSessionId = getRandomString(8)
 
 // Only log status for studio users
-const logStatusEnable = getAllowStudio()
+const logStatusEnable = true // getAllowStudio() HACK: this needs to be setup to be reactive if this is wanted to work correctly
```

Comment:
This is a regression - we are using this logging to trace installation errors for the key clients we have. This hack making this feature essentially useless.

Reply:
I left this with the intention of figuring out to make it reactive, but I'm not sure if doing so would still fulfil the purpose, so I've reverted it to still check localstorage. 
That will keep the behaviour of today, where it will be a bit broken if using 'proper' auth.